crates/vm/levm/src/gas_cost.rs

```diff
@@ -180,27 +180,31 @@ pub const MODEXP_STATIC_COST: u64 = 0;
 pub const MODEXP_DYNAMIC_BASE: u64 = 200;
 pub const MODEXP_DYNAMIC_QUOTIENT: u64 = 3;
 
-pub fn exp(exponent: U256) -> Result<U256, OutOfGasError> {
+pub fn exp(exponent: U256) -> Result<u64, VMError> {
```

Comment:
`OutOfGasError` was a good fit for this function, it says explicitly that it can fail only because of that. Doesn't matter much though.

Reply:
Hi Jere, good morning. Now I remember why I changed the error type. 
It's because of this casting that's  below (The one that returns "VeryLargeNumber")